whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/inetnum/GeofeedValidator.java

```diff
@@ -0,0 +1,59 @@
+package net.ripe.db.whois.update.handler.validator.inetnum;
+
+import com.google.common.collect.ImmutableList;
+import net.ripe.db.whois.common.ip.Ipv4Resource;
+import net.ripe.db.whois.common.ip.Ipv6Resource;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.update.domain.Action;
+import net.ripe.db.whois.update.domain.PreparedUpdate;
+import net.ripe.db.whois.update.domain.UpdateContext;
+import net.ripe.db.whois.update.domain.UpdateMessages;
+import net.ripe.db.whois.update.handler.validator.BusinessRuleValidator;
+import org.springframework.stereotype.Component;
+
+import static net.ripe.db.whois.common.rpsl.AttributeType.GEOFEED;
+import static net.ripe.db.whois.common.rpsl.AttributeType.INET6NUM;
+import static net.ripe.db.whois.common.rpsl.AttributeType.INETNUM;
+
+@Component
+public class GeofeedValidator implements BusinessRuleValidator {
+
+    private static final ImmutableList<Action> ACTIONS = ImmutableList.of(Action.CREATE, Action.MODIFY);
+    private static final ImmutableList<ObjectType> TYPES = ImmutableList.of(ObjectType.INETNUM, ObjectType.INET6NUM);
+    private static final int IPV4_MAXIMUM_PREFIX_SIZE = 24;
+    private static final int IPV6_MAXIMUM_PREFIX_SIZE = 32;
```

Comment:
Legal draft text is changing IPv6 limit from 32 to 48

Reply:
Changed, and changed tests as well.